stripe/src/main/java/com/stripe/android/paymentsheet/AddButtonViewState.kt

```diff
@@ -0,0 +1,14 @@
+package com.stripe.android.paymentsheet
+
+import com.stripe.android.paymentsheet.model.PaymentSelection
+
+/** TODO: Rename to ViewState if can get in the right package */
+sealed class AddButtonViewState {
```

Comment:
This should be `internal`. Also can we consolidate to a single `ViewState` type?

Or maybe:

```
sealed class ViewState {
  sealed class Add : ViewState() {
    // Ready, Confirming, Completed
  }

  sealed class Buy : ViewState() {
    // Ready, Confirming, Completed
  }
}